hpx/parallel/executors/executor_traits.hpp

```diff
@@ -129,14 +132,14 @@ namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v3)
         struct apply_helper
         {
             template <typename Executor, typename F>
-            static auto call(wrap_int, Executor& exec, F && f) -> void
+            static void call(wrap_int, Executor& exec, F && f)
             {
                 exec.async_execute(std::forward<F>(f));
             }
 
             template <typename Executor, typename F>
             static auto call(int, Executor& exec, F && f)
-                -> decltype(exec.apply_execute(std::forward<F>(f)))
+            ->  decltype(exec.apply_execute(std::forward<F>(f)))
```

Comment:
We need to write it this way because this decltype() is used to SFINAE-out the function if the executor (`exec`) does not implement `apply_execute`